WcaOnRails/app/inputs/date_picker_input.rb

```diff
@@ -3,10 +3,10 @@ def input(wrapper_options)
     set_html_options
     set_value_html_option
 
-    template.content_tag :div, class: 'input-group date datetimepicker' do
+    #template.content_tag :div, class: 'input-group date datetimepicker' do
       input = super(wrapper_options) # leave StringInput do the real rendering
-      input_button + input + utc_addon
-    end
+    #  input_button + input + utc_addon
```

Comment:
EDIT
I got it working with the UTC addon in the right place (the rendered input is a datetime_picker here, not a date_picker as I set in application.js...just added a couple lines there to make both work).

I still like the note about the registrations, if only for consistency. (or remove the one in polls/edit)